doc/Pull Request Acceptance Checklist.md

```diff
@@ -0,0 +1,15 @@
+# Pull Request Acceptance Checklist
+
+## Beagle Project
+ * detailed expanatory Javadoc
+ * No checkstyle violations (errors or warnings)
+ * 100% Code coverage by JUnit Tests
+ * JUnit test check were created by another person and are complete (check everything possible)
```

Comment:
Hard to say. I doubt there is a waterproof definition. Maybe we should just state that "no reviewer can think of any additional, meaningful test case". 

Am 23.12.2015 7:39 nachm. schrieb acBerger <notifications@github.com>:

In doc/Pull Request Acceptance Checklist.md:

> @@ -0,0 +1,15 @@ > +# Pull Request Acceptance Checklist > + > +## Beagle Project > + * detailed expanatory Javadoc > + * No checkstyle violations (errors or warnings) > + * 100% Code coverage by JUnit Tests > + * JUnit test check were created by another person and are complete (check everything possible) 

Okay, I see.
Are you able to describe what is meant, so we can replace this misunderstanding sentence?

—
Reply to this email directly or view it on GitHub.